src/System.Security.Cryptography.X509Certificates/tests/CollectionImportTests.cs

```diff
@@ -293,5 +300,57 @@ public static void ImportPkcs12File_Chain_VerifyContents()
                 Assert.Equal(expectedHasPrivateKeys, actualHasPrivateKeys);
             }
         }
+
+
+        [Fact]
+        public static void InvalidStorageFlags()
+        {
+            X509Certificate2Collection coll = new X509Certificate2Collection();
+            byte[] nonEmptyBytes = new byte[1];
+
+            Assert.Throws<ArgumentException>(
+                "keyStorageFlags",
+                () => coll.Import(nonEmptyBytes, string.Empty, (X509KeyStorageFlags)0xFF));
+
+            Assert.Throws<ArgumentException>(
+                "keyStorageFlags",
+                () => coll.Import(string.Empty, string.Empty, (X509KeyStorageFlags)0xFF));
+            
+            // No test is performed here for the ephemeral flag failing downlevel, because the live
+            // binary is always used by default, meaning it doesn't know EphermalKeySet doesn't exist.
+        }
+
+#if netcoreapp11
+        [Fact]
+        public static void InvalidStorageFlags_PersistedEphemeral()
+        {
+            const X509KeyStorageFlags PersistedEphemeral =
+                X509KeyStorageFlags.EphemeralKeySet | X509KeyStorageFlags.PersistKeySet;
+
+            byte[] nonEmptyBytes = new byte[1];
+            X509Certificate2Collection coll = new X509Certificate2Collection();
+
+            Assert.Throws<ArgumentException>(
+                "keyStorageFlags",
+                () => coll.Import(nonEmptyBytes, string.Empty, PersistedEphemeral));
+
+            Assert.Throws<ArgumentException>(
+                "keyStorageFlags",
+                () => coll.Import(string.Empty, string.Empty, PersistedEphemeral));
+        }
+#endif
```

Comment:
These felt related to the other content in this file (testing X509Certificate2Collection.Import), and hopefully the #if can be removed during post-netstandard2.0 work.  It feels particularly relevant given the theory driven by a #if in a couple more lines (it seems pretty bad to me to duplicate the whole file over that)